clients/privacy-center/packages/fides-consent/src/lib/cookie.ts

```diff
@@ -1,17 +1,53 @@
+import { v4 as uuidv4 } from "uuid";
 import { getCookie, setCookie, Types } from "typescript-cookie";
+
 import { ConsentConfig } from "./consent-config";
 import { ConsentContext } from "./consent-context";
 import { resolveConsentValue } from "./consent-value";
 
 /**
- * A mapping from the cookie keys (configurable strings) to the resolved consent value.
+ * Store the user's consent preferences on the cookie, as key -> boolean pairs, e.g.
+ * {
+ *   "data_sales": false,
+ *   "analytics": true,
+ *   ...
+ * }
  */
 export type CookieKeyConsent = {
   [cookieKey: string]: boolean | undefined;
 };
 
-export const CONSENT_COOKIE_NAME = "fides_consent";
+/**
+ * Store the user's identity values on the cookie, e.g.
+ * {
+ *   "fides_user_device_id": "1234-",
+ *   "email": "jane@example.com",
+ *   ...
+ * }
+ */
+export type CookieIdentity = Record<string, string>;
 
+/**
+ * Store metadata about the cookie itself, e.g.
+ * {
+ *   "version": "0.9.0",
+ *   "createdAt": "2023-01-01T12:00:00.000Z",
+ *   ...
+ * }
+ */
+export type CookieMeta = Record<string, string>;
+
+export interface FidesCookie {
+  consent:  CookieKeyConsent;
+  identity: CookieIdentity;
+  fides_meta: CookieMeta;
+};
```

Comment:
This is the updated cookie format- whereas before it was a single object like `{ data_sales: false }` it's now a structured format like `{ consent: { data_sales: false }, identity: { ... }, fides_meta: { ... }}`.

This makes this change pretty forward-compatible with anything we build for privacy experiences, IMO, instead of creating a temporary new cookie that we'd need to keep legacy support for.